pkg/cmd/generate/mock_test.go

```diff
@@ -67,9 +67,21 @@ func newAdminManifestsContext(t *testing.T, config *assets.KubeSawAdmins, files
 
 // ClusterContext part
 
-func newFakeClusterContext(adminManifestsContext *adminManifestsContext, clusterType configuration.ClusterType) *clusterContext {
-	return &clusterContext{
+func newFakeClusterContext(adminManifestsContext *adminManifestsContext, clusterType configuration.ClusterType, options ...fakeClusterContextOption) *clusterContext {
+	ctx := &clusterContext{
 		adminManifestsContext: adminManifestsContext,
 		clusterType:           clusterType,
 	}
+	for _, modify := range options {
+		modify(ctx)
+	}
+	return ctx
+}
+
+type fakeClusterContextOption func(ctx *clusterContext)
+
+func withSpecialKMemberName(specialKMemberName string) fakeClusterContextOption {
```

Comment:
very minor - but the field name seems to be `specific` not `special`, should we use that terminology ?

Reply:
good catch, looks like there is a mix of the formats - unified in f677a45